providers/amazon/provider.yaml

```
@@ -1003,6 +1003,82 @@ config:
         example: '{"Tags": [{"Key": "key", "Value": "value"}]}'
         default: ~
 
+  aws_lambda_executor:
+    description: |
+      This section only applies if you are using the AwsLambdaExecutor in
+      Airflow's ``[core.executor]`` configuration.
+      For more information see:
+      https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda/client/invoke.html
+      https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
+      https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/executors/lambda-executor.html
+    options:
+      conn_id:
+        description: |
+          The Airflow connection (i.e. credentials) used by the Lambda executor to make API calls.
+        version_added: "9.7.0"
```

`9.8.0rc1` is out so it will most likely be released in `9.9.0`

```suggestion
        version_added: "9.9.0"
```

---

Ya fair, this thing is annoying since you're chasing it until it's merged, will update.

---

done